awesome.rb

```diff
@@ -0,0 +1,13 @@
+class Test
+
+  def hello
+    'hello bro'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.